internal/alias/target/public_ids.go

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: BUSL-1.1
+
+package target
+
+import (
+	"context"
+
+	"github.com/hashicorp/boundary/globals"
+	"github.com/hashicorp/boundary/internal/db"
+	"github.com/hashicorp/boundary/internal/errors"
+	"github.com/hashicorp/boundary/internal/types/resource"
+)
+
+func init() {
+	globals.RegisterPrefixToResourceInfo(globals.TargetAliasPrefix, resource.Alias, "alias", Subtype)
```

Comment:
I'm wondering if there should be an internal/alias/alias.go file where "alias" is defined as a const